ibc/store/proofs_ics23.go

```diff
@@ -0,0 +1,159 @@
+package store
+
+import (
+	"crypto/sha256"
+
+	ics23 "github.com/cosmos/ics23/go"
+	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+	"github.com/pokt-network/smt"
+)
+
+type position int
+
+const (
+	left  position = iota // 0
+	right                 // 1
+)
+
+var (
```

Comment:
Should probably mention that this has a direct dependency on constants /values from `pokt-network/smf`